features/step_definitions/node.rb

```diff
@@ -633,9 +633,15 @@
   cb[cb_name] = nodes.select { |n| n.is_worker? }
 end
 
-Given /^I store the number of worker nodes to the#{OPT_SYM} clipboard$/ do |cb_name|
+Given /^I store the number of( linux| windows)? worker nodes to the#{OPT_SYM} clipboard$/ do |os_type, cb_name|
   nodes = BushSlicer::Node.list(user: admin)
-  worker_nodes = nodes.select { |n| n.is_worker? }
+  if !os_type
+    worker_nodes = nodes.select { |n| n.is_worker? }
+  elsif os_type.strip == "linux"
+    worker_nodes = nodes.select { |n| n.is_linux? && n.is_worker?}
+  else
+    worker_nodes = nodes.select { |n| n.is_windows && n.is_worker?}
+  end
```

Comment:
Since we only want the workers, perhaps we can use
```
  worker_nodes = nodes.select { |n| n.is_worker? }
  if os_type =~ /linux/
    worker_nodes = worker_nodes.select { |n| n.is_linux? }
  elsif os_type =~ /windows/
    worker_nodes = worker_nodes.select { |n| n.is_windows? }
  end
```
Not tested though.

Reply:
Good suggestion! updated